docs/edgeql/select.rst

```diff
@@ -843,6 +843,46 @@ shapes just like a non-computed link.
   } filter .name = "Iron Man";
 
 
+Filtering on a known backlink
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another handy use for backlinks is using them to filter and find items
+when doing a ```select`` (or an ``update`` or other operation, of course).
+This can work as a nice shortcut when you have the id of one object that
```

Comment:
I would either style this as "ID" or make it clear you are talking about the property name by making "id" inline code: "`id`"

Whichever you prefer, propagate it to the other instances.

Reply:
Yeah. Ordinarily I'd say backtick but it's such a small word that capitals minus backtick is probably better for visibility so will go with that.